css-module.mjs

```diff
@@ -1,7 +1,7 @@
 // This loader provides a basic facsimile of CSS Modules intended for testing.
-// Use something like esbuild to handle this in production.
+// Use something like esbuild, postcss to handle this in production.
```

Comment:
```suggestion
// Use something like esbuild to handle this in production.
```


I believe the comment was correct before: esbuild handles this for production builds. postcss handles it for dev

Reply:
You last commit un-did the correction 😜